.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+    "trailingComma": "all",
+    "tabWidth": 4,
+    "semi": true,
+    "singleQuote": false,
+    "bracketSpacing": false
+}
```

Comment:
This is only used by jest when formatting tests after inserting inline snapshots.  It is not used for formatting any of the code in `src`.